genesis/src/lib.rs

```diff
@@ -86,6 +157,14 @@ impl RawGenesisBlock {
     pub fn from_path<P: AsRef<Path>>(path: P) -> Result<Self> {
         RawGenesisBlockFile::from_path(path)?.try_into()
     }
+
+    fn unify(self) -> GenesisTransactionBuilder {
+        let mut unified_tx = GenesisTransactionBuilder {
+            isi: vec![Upgrade::new(self.executor).into()],
+        };
+        unified_tx.extend_instructions(self.transactions.iter().flat_map(|v| v.isi.clone()));
```

Comment:
Since genesis block now should have only one transaction, there should not be `transactions` field. It could be possibly renamed to `instructions`.

Reply:
Renamed the field to `transaction` and changes its type.